### PR TITLE
⬆️ updates dictionary version

### PR DESCRIPTION
**Pull request by hlminh2000**

### CODE REVIEW

File: scripts/manage-versions.js [hunk 1]
```diff
@@ -158,9 +159,10 @@ async function runAdd() {
   await fetchAndSaveDiffsForVersion(selectedVersion);
 
   // Update versions file
-  const updatedVersions = currentVersions
-    .concat(selectedVersion)
-    .sort((a, b) => (parseFloat(a) < parseFloat(b) ? 1 : -1));
+  const updatedVersions = orderBy(currentVersions.concat(selectedVersion), (version) => {
+    const [major, minor] = version.split('.');
+    return major * 10 + minor;
```
Comment (joneubank) [inline scripts/manage-versions.js]: works until we have subversions with 3 digits ;)

Comment (joneubank) [reply scripts/manage-versions.js]: we should use sort still but change the comparison, the comparison will first compare the major verison, if the same then compare the subversion.

Comment (hlminh2000) [reply scripts/manage-versions.js]: hmmm I just realize `const [a, b, c] = [1, 2]` will just pass with `c` being `undefined`. Was afraid that'll fail... will update :)

Comment (joneubank) [reply scripts/manage-versions.js]: we aren't using patch versions, just a.b so non issue

**Pull request merged**